internal/collector/collector_test.go

```diff
@@ -1,15 +1,14 @@
-package agent
+package collector
 
 import (
-	"github.com/sreway/yametrics/internal/metrics"
 	"github.com/stretchr/testify/assert"
```

Comment:
goimports && gofumpt!!!

Reply:
[add golangci-lint to github actions](https://github.com/sreway/yametrics/pull/4/commits/1f78daefa7d0bf96839beab6d0a2ea2f64fec31e)